### PR TITLE
Add ps2gl

### PR DESCRIPTION
**Pull request by Wolf3s**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-.PHONY: aalib cmakelibs expat libconfig libid3tag libjpeg_ps2_addons libmad libtap libtiff lua madplay ode romfs sdl sdlgfx sdlimage sdlmixer sdlttf SIOCookie unzip
```
Comment (fjtrujy) [inline Makefile:1]: why have you removed `ode` from here`?

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+.PHONY: aalib cmakelibs expat libconfig libid3tag libjpeg_ps2_addons libmad libtap libtiff lua madplay romfs sdl sdlgfx sdlimage sdlmixer sdlttf SIOCookie ps2stuff ps2gl unzip
 
 all: libraries
-libraries: aalib cmakelibs expat libconfig libid3tag libjpeg_ps2_addons libmad libtap libtiff lua madplay romfs sdl sdlgfx sdlimage sdlmixer sdlttf SIOCookie unzip
+libraries: aalib cmakelibs expat libconfig libid3tag libjpeg_ps2_addons libmad libtap libtiff lua madplay romfs sdl sdlgfx sdlimage sdlmixer sdlttf SIOCookie ps2stuff ps2gl unzip
```
Comment (fjtrujy) [inline Makefile:4]: Additionally these things are sorted alphabetically, so please put in the proper place the `ps2stuff` and `ps2gl`

File: Makefile [hunk 3]
```diff
@@ -75,6 +75,16 @@ ps2_drivers:
 	$(MAKE) -C $@ install
 	$(MAKE) -C $@ clean
 
+ps2stuff:
+	git clone --depth 1 https://github.com/ps2dev/ps2stuff
+	$(MAKE) -C $@ install
+	$(MAKE) -C $@ clean
+
+ps2gl:
```
Comment (fjtrujy) [inline Makefile:83]: if I'm not mistaken, `ps2gl` depends on `ps2stuff` so you need to add that dependency here:
```make
ps2gl: ps2stuff
    git clone --depth 1 https://github.com/ps2dev/ps2gl
    ....
```

File: Makefile (hunk 2 above)
Comment (Wolf3s) [reply Makefile]: Added in alphabetic order thx for the suggestion.

**Comment by Wolf3s** [comment]
@fjtrujy You can merge it.

**Pull request merged**